app/src/main/java/org/sil/storyproducer/controller/MainActivity.kt

```diff
@@ -108,6 +101,33 @@ class MainActivity : BaseActivity(), Serializable {
         }
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        storyPageViewPager.unregisterOnPageChangeCallback(storyPageChangeCallback)
+    }
+
+    var storyPageChangeCallback = object : ViewPager2.OnPageChangeCallback() {
+        override fun onPageSelected(position: Int) {
+            Log.i("MainActivity Story Page", "Selected Tab: $position")
+        }
+    }
+
+    private fun setupStoryPages() {
```

Comment:
how about naming this "setupStoryListTabPages()" or whatever is accurate and a little clearer???

Reply:
Renamed, good idea 👍